sql/burnham_derived/test_result_v1/schema.bq.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "name": "submission_timestamp",
+    "type": "TIMESTAMP",
+    "description": "Time at which this test report was submitted"
```

Comment:
Does BigQuery write this field automatically?

Reply:
No, we should add this in the DAG/burnham logic. It should be an ISO string like "2020-01-01 12:00:00.000" in the json we output.